handbook/engineering/README.md

```diff
@@ -415,6 +415,20 @@ When merging a pull request from a community contributor:
 - Thank and congratulate the contributor.
 - Share the merged PR with the team in the #help-promote channel of Fleet Slack to be publicized on social media. Those who contribute to Fleet and are recognized for their contributions often become great champions for the project.
 
+## Changes to tables' schema
+
+Whenever a PR is proposed for making changes to our [tables' schema](https://fleetdm.com/tables/screenlock)(e.g. to schema/tables/screenlock.yml), it also has to be reflected in our osquery_fleet_schema.json file.
+It should be done by running these commands:
+```
+cd website
+./node_modules/sails/bin/sails.js run generate-merged-schema
+```
+NOTE 1: When adding a new table, make sure it does not already exist with the same name.
+If it does, consider changing the new table name or merge the two tables if it makes sense.
```

Comment:
```suggestion
> When adding a new table, make sure it does not already exist with the same name. If it does, consider changing the new table name or merge the two tables if it makes sense.
```

This will create a call-out box that serves as a note.